src/apps/orders/admin/orders/actions.py

```diff
@@ -90,16 +89,20 @@ def generate_diplomas(modeladmin: Any, request: HttpRequest, queryset: QuerySet)
 @admin.action(description=_("Accept homework"))
 def accept_homework(modeladmin: Any, request: HttpRequest, queryset: QuerySet) -> None:
     studies = Study.objects.filter(order__in=queryset)
-
     studies.update(homework_accepted=True)
 
+    for study in studies:
```

Comment:
```suggestion
    for study in studies.iterator():
```

Это действие происходит в основном потоке, почему бы не сделать его быстрее.